app/views/shared/authentication/_forem_creator_signup.html.erb

```diff
@@ -64,10 +64,10 @@
             Password
             <span class="crayons-field__required crayons-tooltip"></span>
           <% end %>
-          <span class="js-creator-password-visibility">
-            <%= inline_svg_tag("eye.svg", class: "absolute bottom-2 right-2 cursor-pointer js-eye", aria: true, title: "Eye icon") %>
-            <%= inline_svg_tag("eye-off.svg", class: "absolute bottom-2 right-2 cursor-pointer hidden js-eye-off", aria: true, title: "Eye Off icon") %>
-          </span>
+          <button class="crayons-btn crayons-btn--ghost crayons-btn--s js-creator-password-visibility m-0 flex" aria-label="Toggle password visibility">
+            <%= inline_svg_tag("eye.svg", class: "crayons-icon js-eye", aria_hidden: true, title: "Show password") %>
+            <%= inline_svg_tag("eye-off.svg", class: "crayons-icon hidden js-eye-off", aria_hidden: true, title: "Hide password") %>
+          </button>
```

Comment:
This is working so much better - nice one! ✨ 

I'd suggest a couple of final tweaks to make it more accessible:
- Make it a toggle button by adding `aria-pressed="false"` here, and then in the JS when you set the password to visible change the attr to "true" (& back to "false when hidden again). This lets a screen reader user know that it's currently in an active or inactive state 🙂 
- Change the aria-label to "Show password" so it reflects the state that active/inactive (aria-pressed="true"/"false") refers to

I also noticed the change to a button has popped it into a new location - might need a bit of CSS magic to realign it to the input:

![screenshot showing the button occupying the full width above the input](https://user-images.githubusercontent.com/20773163/130204476-3f87f354-2822-4d3b-b381-86de1ecb1c62.png)

Reply:
hard refresh didn't help?